src/compiler/cpp_generator.h

```diff
@@ -64,8 +64,15 @@ struct Parameters {
   grpc::string grpc_search_path;
 };
 
+// A common interface for objects having comments in the source.
+struct CommentHolder {
```

Comment:
nit: I don't like the CommentHolder name much. Can we come up with something better?